### ISSUE TITLE
Keycloak 24

### ISSUE THREAD

**Issue by rblaine95** [adorsys/keycloak-config-cli#996]
### Problem Statement

Keycloak 24 is not currently supported

### Proposed Solution

- Add support for Keycloak 24

### Environment

- Keycloak Version: `24.0.1`
- keycloak-config-cli Version: `5.10.0-23.0.1`

### Additional information

```
2024-03-05 12:36:03.181 ERROR 1 --- [           main] d.a.k.config.KeycloakConfigRunner        : jakarta.ws.rs.ProcessingException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "maxTemporaryLockouts" (class org.keycloak.representations.idm.RealmRepresentation), not marked as ignorable (139 known properties: "userFederationMappers", "rememberMe", "duplicateEmailsAllowed", "adminEventsDetailsEnabled", "users", "clientOfflineSessionMaxLifespan", "webAuthnPolicyRequireResidentKey", "webAuthnPolicyPasswordlessAvoidSameAuthenticatorRegister", "components", "otpPolicyType", "accessCodeLifespanUserAction", "id", "webAuthnPolicyAttestationConveyancePreference", "enabledEventTypes", "applications", "webAuthnPolicyPasswordlessSignatureAlgorithms", "eventsListeners", "ssoSessionMaxLifespanRememberMe", "defaultDefaultClientScopes", "webAuthnPolicyPasswordlessCreateTimeout", "clientOfflineSessionIdleTimeout", "notBefore", "publicKey", "smtpServer", "clientPolicies", "resetPasswordAllowed", "webAuthnPolicyAvoidSameAuthenticatorRegister", "accessTokenLifespanForImplicitFlow", "webAuthnPolicyPasswordlessUserVerificationRequirement", "clientScopes", "internationalizationEnabled", "defaultRole", "accessTokenLifespan", "passwordCredentialGrantAllowed", "federatedUsers", "applicationScopeMappings" [truncated]])
 at [Source: (org.jboss.resteasy.specimpl.AbstractBuiltResponse$InputStreamWrapper); line: 1, column: 1281] (through reference chain: org.keycloak.representations.idm.RealmRepresentation["maxTemporaryLockouts"])
```

**Comment by jonasvoelcker** [comment]
This issue should be fixed when merging this one: https://github.com/adorsys/keycloak-config-cli/pull/998

**Comment by tanadeau** [comment]
I think this can be closed now with the release of 5.11.1.

**Comment by rblaine95** [comment]
That it is, thank you @jonasvoelcker!
